dev-requirements.txt

```
@@ -38,3 +38,6 @@ cryptography~=44.0
 
 # install to verify it doesn't cause a warning (ML-8951)
 evidently~=0.6.2
+
+coverage~=7.8
```

From the previous comment:
```suggestion
coverage[toml]~=7.8
```